utils/threshold_cross_alert.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"log"
+
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/robfig/cron/v3"
+)
+
+// This function runs every 30 seconds to check if unclaimed fees exceeds $1000
+func RunCronJob(bot *tgbotapi.BotAPI, gethUrl string) error {
```

Comment:
This func doesn't actual run the cron job it runs the scheduler for the cron job. Please rename it to something like "StartScheduler" if it is unblocking, or "RunScheduler" if it is blocking